### ISSUE TITLE
[Bug Report] Studios filter in combination with missing urls filter not working

### ISSUE THREAD

**Issue by niemands** [stashapp/stash#867]
**Describe the bug**
When using the `Is Missing is url` filter in combination with `Studios includes studioName` the studio filter gets ignored (I get all scenes with missing urls). This only happens with the missing url filter, when I'm using i.e. the `Is Missing is details` filter I only get results from the specified studio

**To Reproduce**
1. Go to Scenes
2. Add filter `Is Missing is url`
3. Add filter `Studios includes studioName`
4. All scenes with missing urls are shown, even from studios not included in the filter

**Expected behavior**
Only scenes from the given studios are shown

**Stash Version:** v0.3.0-38-gcf003a5

**Desktop:**
 - OS: Win10
 - Browser: Firefox
 - Version: 81.0.2

**Additional context**
The same problem occurs with the excluding studios filter

**Comment by DampToast** [comment]
I tested this today on 0.4.0 release and was unable to reproduce this.

**Comment by niemands** [comment]
That's weird. I just have testet it again on version `v0.4.0-16-g6eea33a` (latest dev) and I still have this issue. The same issue occurs when calling the GraphQL directly. 

#### Query:
Searching for Studio id 53:
```
query {
    findScenes(
        scene_filter: {is_missing: "url" studios: {value: [53], modifier: INCLUDES}}
        filter: { per_page: 2}
    ) {
        count
        scenes {
            id
            url
      		studio {id}
        }
    }
}
```

#### Result:
Getting results with studio id 51, changing `INCLUDES` to `INCLUDES_ALL` makes no differenece in the result
```
{
  "data": {
    "findScenes": {
      "count": 33,
      "scenes": [
        {
          "id": "925",
          "url": null,
          "studio": {
            "id": "51"
          }
        },
        {
          "id": "770",
          "url": null,
          "studio": {
            "id": "51"
          }
        }
      ]
    }
  }
}
```

**Comment by DampToast** [comment]
I think I found the bug although I don't know the fix.

First, INCLUDES_ALL is not valid for Scenes so that's why that one didn't work.

But, I did figure out that if you do a similar query with tags or performers you get incorrect results if you use INCLUDES or EXCLUDES but not if you use INCLUDES_ALL.  So I think there is an issue with the way INCLUDES and EXCLUDES are working regardless of if you are querying studio, tags, or performers.
